ReactWindows/ReactNative.Shared/Bridge/IReactInstance.cs

```diff
@@ -60,5 +61,15 @@ public interface IReactInstance : IAsyncCancelable
         /// <typeparam name="T">Type of native module.</typeparam>
         /// <returns>The native module instance.</returns>
         T GetNativeModule<T>() where T : INativeModule;
+
+        /// <summary>
+        /// Fires when there were, but are now no, pending bridge calls
```

Comment:
>Fires when there were, but are now no [](start = 12, length = 37)

"Fires when the number of pending bridge calls goes to zero."

Reply:
Just suggestions, the comments are a bit confusing.

---
In reply to: [186073353](https://github.com/Microsoft/react-native-windows/pull/1726#discussion_r186073353) [](ancestors = 186073353)